api.yaml

```
@@ -0,0 +1,363 @@
+openapi: 3.0.0
+info:
+  title: Content Management API
+  description: API for managing content and content types.
+  version: 1.0.0
+servers:
+- url: https://api.everest.com/v1
+
+paths:
+  /content-types:
+    get:
+      summary: List all Content Types
+      responses:
+        "200":
+          description: A list of content types
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                    name:
+                      type: string
+                      example: "my-content-type"
+
+    post:
+      summary: Create a new Content Type
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                name:
+                  type: string
+                  example: "Blog Post"
+                schema:
+                  type: string
```

O esquema será um objeto do tipo ContentTypeSchema (ou algo semelhante). Mas, considerando que ainda não sabemos como esse tipo será, que tal definir como `type: object` por enquanto?

```suggestion
                schema:
                  type: object
```